ui/sortable.js

```diff
@@ -110,19 +111,19 @@ return $.widget("ui.sortable", $.ui.mouse, {
 	},
 
 	_setHandleClassName: function() {
-		this.element.find( ".ui-sortable-handle" ).removeClass( "ui-sortable-handle" );
+		var that = this;
+		this._removeClass( this.element.find( ".ui-sortable-handle" ), "ui-sortable-handle" );
 		$.each( this.items, function() {
-			( this.instance.options.handle ?
-				this.item.find( this.instance.options.handle ) : this.item )
-				.addClass( "ui-sortable-handle" );
+			that._addClass(
+				this.instance.options.handle ?
```

Comment:
This threw me off at first. Okay to ignore, but it should really use the `each` arguments, not `this`.